### DISCUSSION TITLE
Incomplete comparison result

### DISCUSSION THREAD

**Discussion by adrianmichel** [WinMerge/winmerge#2202] [Q&A]
I ran a comparison on two folders (which are identical btw). During the comparison process, WinMerge window lists all the files. After the comparison ends, the windows only shows 10 largest files (regardless of the sorting criteria - they could be sorted by extension or anything else, it's still the largest 10 file).

In the following screenshots, notice the number of files indicated in the lower right corner for each of them.

![image](https://github.com/WinMerge/winmerge/assets/31669083/9eda76c8-148c-4881-a362-187a10ccdc45)
![image](https://github.com/WinMerge/winmerge/assets/31669083/64bcbe72-6178-4396-8401-981cfe4e8d05)

I tried to find filters that would limit the results and lead to this output but couldn't find any (that would apply to this case anyway).

Any suggestions?

**Comment by adrianmichel** [comment]
Checking "Show binary files" resolved the problem. It's unclear however why some of the media files are considered binary and others are not:
![image](https://github.com/WinMerge/winmerge/assets/31669083/a947e0dc-57ba-450b-a7f1-348bfe78f519)

**Comment by sdottaka** [comment]
By default, WinMerge switches the comparison method to the Binary Contents compare method for faster comparisons when the file size exceeds 64MB. The Binary Contents compare method does not determine whether a file is a binary file, so it seems that files larger than 64MB were not determined to be binary files.

![image](https://github.com/WinMerge/winmerge/assets/98126/25f38a09-126d-45a7-87b6-d5cd748b6258)

**Comment by adrianmichel** [comment]
This seems like an inconsistency - the comparison algorithm should not determine whether a file is of binary type or not. But at least now I know what's going on.

Thanks.
